src/features/list-view/ListView.js

```diff
@@ -32,6 +32,12 @@ type CellRendererArgs = {|
     style: Object,
 |};
 
+type ComputeColumnWidthArgs = {|
```

Comment:
Could we move this and CellRendererArgs to a flowType.js file?

Reply:
Glad you pointed it out! I'll do that.

Reply:
Done.